manual/parts/cspace.tex

```diff
@@ -65,7 +65,7 @@ \subsection{CSpace Creation}
 Like any other object, a \obj{CNode} must be created by calling
 \apifunc{seL4\_Untyped\_Retype}{untyped_retype} on an appropriate
 amount of untyped memory (see \autoref{sec:object_sizes}).  The caller
-must therefore have a capability to enough untyped memory as well as
+must therefore have a capability to ensure enough untyped memory as well as
```

Comment:
I think "ensure" this should not be added here, or there sentence needs more rewording. You need *one* untyped mem cap that is at least the size that is needed for the c-space

Reply:
You’re right, ensure is not the right word. Maybe “a capability to sufficiently large untyped memory as well as..”